linkerd/app/inbound/src/policy/http.rs

```diff
@@ -171,6 +171,8 @@ where
             }
         };
 
+        try_fut!(self.check_rate_limit());
```

Comment:
It would be good to exercise this in a [test](https://github.com/linkerd/linkerd2-proxy/blob/b580e657e565d77e4138c34397fea7182fd1797b/linkerd/app/inbound/src/policy/http/tests.rs).